pkg/cronjob/es_index_cleaner.go

```diff
@@ -17,6 +17,18 @@ func CreateEsIndexCleaner(jaeger *v1alpha1.Jaeger) *batchv1beta1.CronJob {
 	esUrls := getEsHostname(jaeger.Spec.Storage.Options.Map())
 	trueVar := true
 	name := fmt.Sprintf("%s-es-index-cleaner", jaeger.Name)
+
+	var envFromSource []v1.EnvFromSource
+	if len(jaeger.Spec.Storage.SecretName) > 0 {
+		envFromSource = append(envFromSource, v1.EnvFromSource{
+			SecretRef: &v1.SecretEnvSource{
+				LocalObjectReference: v1.LocalObjectReference{
+					Name: jaeger.Spec.Storage.SecretName,
```

Comment:
@jpkrohling I am a bit outdatted with represents secretName in the storage spec?

Reply:
@pavolloffay I'm guessing its https://github.com/jaegertracing/jaeger-operator#secrets-support